.github/workflows/cd.yml

```diff
@@ -1,9 +1,15 @@
 name: Deploy
 
 on:
-  # !! Uncomment the following lines once you've set up the dev environment and ready to turn on continuous deployment
-  # push:
-  #   branches: ["main"]
```

Comment:
don't uncomment this. this workflow won't work on the template since the template doesn't have any resources deployed.